scripts/scc_cocomo.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
When I execute the script it doesn't output anything which doesn't appear to be intended. I also found the aforementioned error when I try to run line 15 directly.